src/App.js

```diff
@@ -1,5 +1,90 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+
+  getRandomNumber(){
+    const computerNumber = [];
+    while (computerNumber.length < 3){
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if(!computerNumber.includes(number))
+        computerNumber.push(number);
+    }
+    return computerNumber;
+  }
```

Comment:
Set 객체가 includes 함수에 비해 중복 제거에 효과적이에요!
```suggestion
  getRandomNumber() {
    const computer = new Set();
    while (computer.size < 3) {
      const number = Random.pickNumberInRange(1, 9);
      computer.add(number);
    }

    return [...computer];
  }
```